Add meter (#395)
subquery/documentation: docs/quickstart/quickstart_chains/meter.md

> # Scroll (Sepolia Testnet) Quick Start

Reviewer: ```suggestion
# Meter Quick Start
```

## Goals

The goal of this quick start guide is to index all transfers and approval events from the [Wrapped Eth](https://scan.meter.io/address/0x983147fb73a45fc7f8b4dfa1cd61bdc7b111e5b6) on [Meter Network](https://scan.meter.io).

::: warning

> Before we begin, **make sure that you have initialised your project** using the provided steps in the [Start Here](../quickstart.md) section. Please initialise an a Scroll Sepolia project.

Reviewer: ```suggestion
Before we begin, **make sure that you have initialised your project** using the provided steps in the [Start Here](../quickstart.md) section. Please initialise an a Meter project.
```

:::

In every SubQuery project, there are [3 key files](../quickstart.md#_3-make-changes-to-your-project) to update. Let's begin updating them one by one.

::: tip Note
The final code of this project can be found [here](https://github.com/subquery/ethereum-subql-starter/blob/main/Meter/meter-starter).

> We use Ethereum packages, runtimes, and handlers (e.g. `@subql/node-ethereum`, `ethereum/Runtime`, and `ethereum/*Hander`) for Scroll. Since Scroll is an EVM-compatible layer-2 scaling solution, we can use the core Ethereum framework to index it.

Reviewer: ```suggestion
We use Ethereum packages, runtimes, and handlers (e.g. `@subql/node-ethereum`, `ethereum/Runtime`, and `ethereum/*Hander`) for Meter. Since Meter is an EVM-compatible layer-2 scaling solution, we can use the core Ethereum framework to index it.
```
